setup.py

```diff
@@ -34,13 +29,17 @@
   long_description_content_type='text/markdown',
   url='https://github.com/google/shaka-streamer',
   packages=setuptools.find_packages(),
-  install_requires=install_prerequisites,
+  install_requires=[
+      'PyYAML',
+      'pywin32;platform_system=="Windows"',
+  ],
   scripts=['shaka-streamer'],
   classifiers=[
       'Programming Language :: Python :: 3',
       'License :: OSI Approved :: Apache Software License',
       'Operating System :: POSIX :: Linux',
       'Operating System :: MacOS :: MacOS X',
+      'Operating System :: Microsoft :: Windows',
```

Comment:
Should we remove the OS classifiers now since we already support the three of them?

Reply:
I can't find any docs from pypi indicating one way or the other is best.  Let's leave them for now.  Thanks!